tidb-binlog/pump_client/client.go

```diff
@@ -303,11 +310,6 @@ func (c *PumpsClient) WriteBinlog(binlog *pb.Binlog) error {
 				break
 			}
 
-			if isConnUnAvliable(err) {
```

Comment:
maybe we can retry directly on this pump if err is `Unavailable`

Reply:
if restart TiDB, the connection is unavailable, can you sure the connection will recover by itself?

Reply:
if reset pump, the connection is unavailable, can you sure the connection will recover by itself?

Reply:
ClientConn will
maintain the connections to server and reconnect if need

Reply:
ClientConn will
maintain the connections to server and reconnect if need